seldonite/analyze.py

```diff
@@ -51,7 +51,12 @@ def countries(x):
                             liste.append(country)
                         return liste
 
-            udfCountry = udf(countries, StringType())
-            df = df.withColumn('countries', udfCountry(df.all_text))
+                udfCountry = udf(countries, ArrayType(StringType(), True))
+                df = df.withColumn('countries', udfCountry(df.all_text))
 
-            #TODO: need a function to count the country proportions
\ No newline at end of file
+                
+                key = df.select(explode(df.countries).alias("key"))
+                df_with_key = df.withColumn("key", lit(key))
+
+                # TODO: what should I return?
+                key.groupBy(col("key")).count().show()
```

Comment:
Here `key.groupBy(col("key")).count()` is a dataframe itself, so just return that. This will work if you use the new `Runner` class system I have set up.